vendor/k8s.io/kubernetes/pkg/kubectl/cmd/config/config_test.go

```diff
@@ -87,6 +87,7 @@ func TestCurrentContext(t *testing.T) {
 }
 
 func TestSetCurrentContext(t *testing.T) {
+	t.Skip()
```

Comment:
document in the commit message why

Reply:
> document in the commit message why

removed that commit.  Found the lock file culprit